### PR TITLE
Editing all comments as per 903

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3491
Also see: https://github.com/hmcts/rpx-xui-webapp/pull/903

### Change description ###
Editing code as per Paul's PR comments

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/workAllocation/index.ts [hunk 1]
```diff
@@ -109,14 +103,17 @@ export async function getAllCaseWorkers(req: EnhancedRequest, res: Response, nex
 }
 
 export async function retrieveAllCaseWorkers(req: EnhancedRequest, res: Response): Promise<Caseworker[]> {
+  if (req.session && req.session.caseworkers) {
+    return req.session.caseworkers;
```
Comment (paul-graham) [inline api/workAllocation/index.ts:107]: Shouldn't this return a `Promise` with a `Caseworker[]` payload? 🤔

Comment (paul-graham) [reply api/workAllocation/index.ts:107]: Oh, no, ignore that - I hadn't appreciated that the `async` will automatically wrap it in a `Promise` if it doesn't return one. Cool cool cool. 😄

**Review by paul-graham — COMMENTED** [review]
All the changes in here look good. However, there are still two conversations on https://github.com/hmcts/rpx-xui-webapp/pull/903 that I'd like further clarity on. They're minor, but I'd prefer a resolution.

**Review by paul-graham — APPROVED** [review]
Happy with these changes. Thank you, @connorpgpmcelroy. 👍

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc

**Pull request merged**